src/components/ClaimForm.js

```diff
@@ -193,7 +198,7 @@ class ClaimForm extends Component {
           }
         }
         if (!this.props.forReview) services.pop();
-        if (services.length && services.filter((s) => !this.canSaveDetail(s, "service")).length) {
+        if (services.length && services.filter((s) => !this.canSaveDetail(s, "service", forReview=forReview)).length) {
```

Comment:
It should be enough to pass forReview here, no need to pass forReview=forReview.